tests/console/ipsec_tool.pm

```diff
@@ -0,0 +1,95 @@
+# SUSE's rsyslog tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Test Case 1595178: FIPS: ipsec-tool
+# Test Racoon host-to-host scenario in fips mode
```

Comment:
Suggest to remove "Test Case 1595178: FIPS: ipsec-tool", just type "Summary: Test Racoon host-to-host scenario".
The script can also be used for non-fips test.